geth/jail/handlers.go

```diff
@@ -89,6 +89,20 @@ func registerHandlers(jail *Jail, cell *JailCell, chatID string) error {
 	return nil
 }
 
+// makeAsyncSendHandler returns jeth.sendAsync() handler.
+func makeAsyncSendHandler(jail *Jail) func(call otto.FunctionCall) (response otto.Value) {
+	return func(call otto.FunctionCall) (response otto.Value) {
+		go func() {
+			res := jail.Send(call)
+
+			// Deliver response to provided callback.
+			newResultResponse(call.Otto, res)
+		}()
+
+		return otto.UndefinedValue()
+	}
```

Comment:
Well, I tried. I did:
```go
go func(){
   fmt.Println("I'm invoked!")
}()
```
inside the handler and at its initiation for 2 different runs and only in 1 case had the response received. Could you double-check this, please?